frontend/src/components/VSearchOption.vue

```diff
@@ -32,6 +32,12 @@ export default {
     return {
       selected: null
     }
+  },
+
+  watch: {
+    selected (val) {
+      this.$emit('change-option', val)
```

Comment:
I think you can just emit on input, and then get the value with v-model where you use the v-search-option component? I.e. no need to create a new event

Reply:
It, does'nt work because I use the event as a method